Playbooks/playbook-Dedup_-_Generic_v2.yml

```diff
@@ -0,0 +1,584 @@
+      version: -1
+      name: ML
+      description: ""
+      type: title
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "7"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 950,
+          "y": 340
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "12":
+    id: "12"
+    taskid: 58c2edb1-ada1-4220-844d-805f1da22ae9
+    type: title
+    task:
+      id: 58c2edb1-ada1-4220-844d-805f1da22ae9
+      version: -1
+      name: Rules
+      description: ""
+      type: title
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "6"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 480,
+          "y": 340
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "13":
+    id: "13"
+    taskid: 0eaa279f-7d04-469d-8bcd-0ca1f7942c88
+    type: condition
+    task:
+      id: 0eaa279f-7d04-469d-8bcd-0ca1f7942c88
+      version: -1
+      name: Close duplicates?
+      description: ""
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "14"
+      "yes":
+      - "8"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: inputs.CloseAsDuplicate
+                transformers:
+                - operator: toLowerCase
+            iscontext: true
+          right:
+            value:
+              simple: "yes"
+    view: |-
+      {
+        "position": {
+          "x": 970,
+          "y": 830
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "14":
+    id: "14"
+    taskid: 2f6d5087-c0f1-47e3-8065-06cc25ced46a
+    type: regular
+    task:
+      id: 2f6d5087-c0f1-47e3-8065-06cc25ced46a
+      version: -1
+      name: Link similar incidents
+      description: commands.local.cmd.linkIncidents
+      script: Builtin|||linkIncidents
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "9"
+    scriptarguments:
+      action: {}
+      incidentId: {}
+      linkedIncidentIDs:
+        complex:
+          root: similarIncident
+          accessor: rawId
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 740,
+          "y": 1000
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+view: |-
+  {
+    "linkLabelsPosition": {
+      "1_9_#default#": 0.32,
+      "5_9_#default#": 0.28
+    },
+    "paper": {
+      "dimensions": {
+        "height": 1275,
+        "width": 1570,
+        "x": 10,
+        "y": -40
+      }
+    }
+  }
+inputs:
+- key: DuplicateMethod
+  value: {}
+  required: true
+  description: Select a method for identifying duplicate incidents. Can be "ml", "rules",
```

Comment:
I think It's better to specify a little bit details here:
rules - define specific rules, like similar incident fields & labels. It's the best method if you know the exact logic to find similar incidents.
text - text similarity, based on TF-IDF - unique word frequency in the incidents (based on similar incident fields)
ml - machine learning model, was trained on similar phishing incidents. consider similar labels, incident fields and indicators.

We need some doc help here

Reply:
@erezh31 We always have doc review. @yaron-libman is going over it a bit later today. 

@mayagoldb I agree with Erez, please work together and update description before Yaron's review.